client/src/PublicHome.js

```diff
@@ -59,41 +107,86 @@ class PublicHome extends Component {
           </Jumbotron> 
           :
           <Media left top >
-            <Media 
+            <Media
+              onMouseLeave={()=>{this.setState({logo_hover: false})}} 
+              onMouseEnter={()=>{this.setState({logo_hover: true})}}
               style={{paddingLeft:'5em',paddingTop:'2em',maxWidth:'35em'}} 
-              object src={Logo} alt="Generic placeholder image"/>
+              object src={this.state.logo_hover ? LogoHoverImg: LogoImg} 
+              alt="Civitas Logo"/>
           </Media>}
-
-          <Jumbotron style={{paddingLeft:'5em',backgroundColor:'#FFFFF'}}>
-            <Button outline color="primary" 
+          {this.props.logged_in ? 
+          <Jumbotron style={{paddingLeft:'5em',
+            paddingTop:'6.13em', paddingBottom:'6.13em',backgroundColor:'#FFFFF'}}>
+            <Media className="mt-1">
+              <Link to="/groups">
+              <Button outline color="primary" 
+                onMouseLeave={()=>{this.setState({register_hover: false})}} 
+                onMouseEnter={()=>{this.setState({register_hover: true})}}>
+                <Media middle>
+                  <Media 
+                    style={{maxWidth:'10em'}} 
+                    object src={this.state.register_hover ? 
+                    GroupsHoverImg : GroupsImg} alt="Groups" />
+                </Media>
+              </Button>
+              </Link>
+            </Media>
+          </Jumbotron>
+          :
+          <Jumbotron style={{paddingLeft:'5em',backgroundColor:'#FFFFF',
+          paddingRight:'0'}}>
+            <Button outline color="primary" onClick={this.toggleLogin}

```

Comment:
If logged in, show button linking to groups in public homepage instead of login/register